ts/input/tex/ParseUtil.ts

```diff
@@ -33,668 +33,667 @@ import {entities} from '../../util/Entities.js';
 import {MmlMunderover} from '../../core/MmlTree/MmlNodes/munderover.js';
 
 
-namespace ParseUtil {
```

Comment:
I would like to see this replaced by an object `export const ParseUtil {...}` that has the exported functions as methods and the constants as properties (since it is always used as `ParseUtil.method()` anyway.  Then you can use `import {ParseUtil} from '.../ParseUtil.js';` as with all the other objects, rather than needing `* as ParseUtil`.  It would also give a natural place to put the `addUnit()` that I suggest below, for example.

One reason for this is that it makes it possible to override the various functions, if need be.  That is, if there is a bug in a function, say `fixInitialMO()`, then one could replace `ParseUtil.fixInitialMO()` in the startup configuration with a fixed version until a new MathJax version is released.  But with individually exported functions, that can't the done.

We may want to consider similar changes as we remove namespaces from other files during the code cleanup.